src/users/cd-redirect-to-login.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <div class="cd-redirect-to-login">
+    <p class="cd-redirect-to-login__heading">{{ $t('Have you booked an event before?') }}</p>
+    <div>
+      <router-link :to="login" class="btn btn-secondary cd-redirect-to-login__button">{{ $t('Log into your account') }}</router-link>
+    </div>
+  </div>
+</template>
+
+<script>
+
```

Comment:
More gratuitous whitespace! The linter must be resting. :P

Reply:
I NEED SOME SPACE MAN :p